documentation/release_notes.rst

```diff
@@ -8,6 +8,60 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.4
+=============
+
+New Features
+------------
+-  Added the range-based versions of the following algorithms: ``any_of``, ``adjacent_find``,
+   ``copy_if``,  ``remove_copy_if``, ``remove_copy``, ``none_of``, ``swap_ranges``, ``unique``, ``unique_copy``,
+   ``reverse``, ``reverse_copy``, ``rotate_copy``, ``replace_copy``,  ``replace_copy_if``.
+-  Added non-blocking versions of the following algorithms: ``inclusive_scan``, ``exclusive_scan``,
```

Comment:
Should it be "asynchronous versions"?

Reply:
"non-blocking versions" terminology was used in 2021.2.0 release notes. To change to "asynchronous versions" or to keep consistent?

Reply:
I have a slight favor for non-blocking to describe the behavior as an attribute. How about changing the description to asynchronous algorithm? But we should make the change consistent to the change log.

Reply:
@reble changes were added. Feel free to review